tests/replicator.rs

```diff
@@ -45,25 +44,27 @@ fn test_replicator_startup() {
         tmp_copy_ledger(&leader_ledger_path, "replicator_test_validator_ledger");
 
     {
-        let (signer_service, signer) = LocalVoteSignerService::new();
-        let signer_proxy = VoteSignerProxy::new(&leader_keypair, signer);
-        let vote_account_id = signer_proxy.vote_account.clone();
+        let signer_proxy = VoteSignerProxy::new(
```

Comment:
I was thinking this would read:

```rust
let vote_signer = LocalVoteSigner::new(&leader_keypair);
```

Because this is a unit test, we create the "local" version of the object that implements the `VoteSigner` trait.

Reply:
It is creating a local version of the VoteSigner trait. ```VoteSignRequestProcessor``` implements VoteSigner trait.

Reply:
Maybe I can rename VoteSignRequestProcess to LocalVoteSigner. That would convey the meaning properly.

Also, I think VoteSignerProxy should be renamed. It's a wrapper around vote signing (not a proxy anymore). Any suggestions?